include/rfb.js

```diff
@@ -121,6 +123,9 @@ var RFB;
         this._viewportDragPos = {};
         this._viewportHasMoved = false;
 
+        // QEMU Extended Key Event support - default to false
+        this._QEMU_Extended_Key_Event = false;
```

Comment:
Good catch, haven't realized I mixed cases here. I will change it to _qemuExtKeyEventSupported in the next pass